api.raml

```diff
@@ -69,6 +92,87 @@ schemas:
         },
         "type": "object"
       }
+  - UserList: |
+      {
+        "$schema": "http://json-schema.org/draft-03/schema",
+        "items": {
+          "properties": {
+              "employee_id": {
+                "required": false,
```

Comment:
No, just that this type of data is not require to have any particular properties. The place this is currently used (`GET /user`) defaults to include `employee_id` and `active`. It is possibly to request a specific set of fields that doesn't include either of those two defaults, though, so even they aren't required for this type of data.